### DISCUSSION TITLE
i get UE4 LegacyVersion: unsupported value -8 when trying to export a .uasset file

### DISCUSSION THREAD

**Discussion by BecoolTJ** [gildor2/UEViewer#300] [Q&A]
call stack listed here:
UE4 LegacyVersion: unsupported value -8
FPackageFileSummary::Serialize4 <- FPackageFileSummary<<: Ver=0/0 <- UnPackage::UnPackage: /Game/Prometheus/Characters/NimbleBlaster/C_NimbleBlaster.uasset, ver=100000/0, game=ue4.0 <- UnPackage::LoadPackage(info): /Game/Prometheus/Characters/NimbleBlaster/C_NimbleBlaster.uasset <- UIMenuItem::HandleCommand <- UIMenu::Popup <- UIBaseDialog::WndProc <- UIBaseDialog::CustomMessageLoop <- UIBaseDialog::ShowDialog: modal=1, title="Choose a package to open" <- CUmodelApp::ShowPackageUI <- Main: umodel_build=1589

anyone know whats going wrong?

**Comment by BecoolTJ** [comment]
ok i just checked and the game im trying to export from seems to be a UE5 game which doesnt seem to be supported

**Comment by gildor2** [comment]
Some UE5 games has custom UModel builds
https://www.gildor.org/smf/index.php/board,44.0.html
